wasmbinding/stargate_whitelist.go

```diff
@@ -37,6 +37,8 @@ var stargateWhitelist sync.Map
 func init() {
 	// cosmos-sdk queries
 
+	setWhitelistedQuery("/cosmos.auth.v1beta1.Query/Account", &authtypes.QueryAccountResponse{})
```

Comment:
Note to self. Where did this come from? Do we need it?

Reply:
Lets remove from this PR, and do in another

Reply:
done. Not sure what that was